packages/@o3r/store-sync/src/sync-storage/storage-sync.ts

```diff
@@ -3,6 +3,9 @@ import { deepFill } from '@o3r/core';
 import type { Logger } from '@o3r/core';
 import type { StorageKeyConfiguration, StorageKeys, SyncStorageConfig, SyncStorageSyncOptions } from './interfaces';
 
+/** The recompute action type of ngrx store devtools */
+const RECOMPUTE = '@ngrx/store-devtools/recompute';
```

Comment:
you could use `RECOMPUTE` from `@ngrx/store-devtools` instead of redefining the variable

Reply:
Would have to add dependency of that package on `o3r-store-sync`. Also, this adds the entire dev tools chunk to the main bundle if we use this method for syncing stores in the app module.

Reply:
A compromise can be to import the type only (to ensure alignment):

```typescript
import type { RECOMPUTE } from '@ngrx/store-devtools';

const RECOMPUTE: typeof RECOMPUTE = '@ngrx/store-devtools/recompute';
```

Reply:
Updated